packages/opentelemetry-sdk-trace-base/test/common/Span.test.ts

```diff
@@ -771,30 +771,6 @@ describe('Span', () => {
     });
   });
 
-  it('should set a link', () => {
```

Comment:
Why remove this test?

Reply:
Described it in the PR description, it's a duplicate of [another test](https://github.com/open-telemetry/opentelemetry-js/pull/4536/files#diff-b7e69d0a91a703e978e4c235c2a5550817a50251fe58d7c9e72245a5c698d4f2L932), but in this case the test had no assertions.